packages/server/tests/api/rest.test.ts

```diff
@@ -1546,6 +1637,54 @@ describe('REST server tests', () => {
 
                     expect(r.status).toBe(404);
                 });
+
+                it('create relation with compound id', async () => {
+                    await prisma.user.create({ data: { myId: 'user1', email: 'user1@abc.com' } });
+                    await prisma.post.create({ data: { id: 1, title: 'Post1' } });
+
+                    const r = await handler({
+                        method: 'post',
+                        path: '/postLike',
+                        query: {},
+                        requestBody: {
+                            data: {
+                                type: 'postLike',
+                                id: `1${idDivider}user1`,
+                                attributes: { userId: 'user1', postId: 1, superLike: false },
+                            },
+                        },
+                        prisma,
+                    });
+
+                    expect(r.status).toBe(201);
+                });
+
+                describe('compound id', () => {
```

Comment:
Is this new level of nesting intentional?

Reply:
Nope, forgot to remove as some tests ended up elsewhere.